src/exercise/client/src/Components/BottomButtons/BottomButtons.js

```diff
@@ -0,0 +1,84 @@
+import "./BottomButtons.css";
+import { Button } from "monday-ui-react-core";
+import { useState } from "react";
+
+// sort tasks by date they created
+function sortTime(ascTime) {
+  const taskSections = document.body.querySelectorAll("div.taskSection");
+  let tmpTaskSections = [...taskSections];
+  tmpTaskSections.sort((a, b) => {
+    a = a.querySelector("div.task").querySelector("p.time").id;
+    b = b.querySelector("div.task").querySelector("p.time").id;
+    if (ascTime) {
+      return new Date(a) - new Date(b);
+    }
+    return new Date(b) - new Date(a);
+  });
+  for (let i = 0; i < tmpTaskSections.length; i++) {
+    taskSections[i].parentNode.appendChild(tmpTaskSections[i]);
+  }
```

Comment:
You don't need and you really want to avoid manipulating the dom like this now that you have react.
you can use that seTasks() in the state of the todoList component in order to reorder your list
just call setTasks() with the right order of the list